src/main/java/edu/kis/vh/nursery/StackInterface.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.vh.nursery;
+
+public interface StackInterface {
+    public static final int EMPTY_STACK_HEAD_INDEX = -1;
```

Comment:
Is it a correct approach to fixing errors showed by tests? This reverted (3.1.9)